### ISSUE TITLE
Improve the keyboard and screen reader accessibility for the main UI

### ISSUE THREAD

**Issue by vick08** [easydiffusion/easydiffusion#1768]
The main UI is not very accessible for keyboard and screen reader users. Specifically, the top menus such as "Generate", "settings", "model tools", etc, cannot be reached by the TAB key and, once expanded, operated on with arrow keys.

**To Reproduce**
1. Open the main UI page, i.e. the UI that starts after running the "start" script of Easy DIffusion.
2. Once the UI loads, use the TAB key to explore the interface and try, accessing the top menus, for example.

**Expected behavior**
The TAB key should allow the user to navigate to each of the menus, ENTER to expand, and up/down arrows to navigate, ENTER to activate  them.

**Desktop (please complete the following information):**
 - OS: Mac OS 14.4.1
 - Browser: Google Chrome
 - Version: 124.0.6367.61 (Official Build) (arm64)

**Additional context**
This website with examples of navigation patterns may be helpful: https://www.w3.org/WAI/ARIA/apg/patterns/
